src/scales/scale.linearbase.js

```diff
@@ -41,8 +42,15 @@ function generateTicks(generationOptions, dataRange) {
 		spacing = Math.ceil(spacing * factor) / factor;
 	}
 
-	niceMin = Math.floor(dataRange.min / spacing) * spacing;
-	niceMax = Math.ceil(dataRange.max / spacing) * spacing;
+	// Loop here is for the case when min / spacing === max / spacing
+	// This happens due to float inccuracy when min and max are really close to each other
+	// One case: min = 1.8548483304974972 and max = 1.8548483304974974
+	do {
+		niceMin = Math.floor(dataRange.min / spacing) * spacing;
+		niceMax = Math.ceil(dataRange.max / spacing) * spacing;
+		spacing *= 2;
+	} while (niceMin === niceMax && ++i < 8);
+	spacing /= 2;
```

Comment:
personally I would prefer removing this line and doing `spacing *= 2` only `if (i > 0)`